### DISCUSSION TITLE
Source?

### DISCUSSION THREAD

**Discussion by FilipeF12** [LykosAI/StabilityMatrix#1017] [Q&A]
Where is the python source for google_blockly.cpython-310.pyc ?

**Comment by ionite34** [comment]
Where are you seeing this file? It's not part of our repository or release files.

**Reply by FilipeF12** [reply]
Is that not part of the release?

**Comment by FilipeF12** [comment]
Here..
C:\stability matrix\Data\Packages\Stable Diffusion WebUI Forge\modules_forge\\\_\_pycache\_\_\
